python/docs/intro.rst

```diff
@@ -0,0 +1,93 @@
+About the project
+=================
+
+Welcome to dlisio. dlisio is a python package for reading Digital Log
+Interchange Standard (DLIS) v1. A version 2 exist, and has been around for
+quite a while, but it is our understanding that most dlis files out there are
+still version 1. Hence dlisio's focus is put on version 1 [1]_, for now.
+
+dlisio is an attempt at abstract away a lot of the pain of dlis and give access
+to the data in a simple and easy-to-use manner. It gives the user the ability
+to work with dlis-files without having to know the details of the standard
+itself. Its main focus is making the data accessible while putting little
+assumptions on how the data is to be used. 
+
+dlisio is written and maintained by Equinor ASA as a free, simple, easy-to-use
+way of reading well logs that can be tailored to our needs, and as a
+contribution to the free software community.
+
+Please note that dlisio is still in alpha, so expect breaking changes between
+versions.
+
+Digital Log Interchange Standard (DLIS)
+=======================================
+
+dlis is a binary file format for well logs, developed by Schlumberger in the
+late 80's and published by the American Petroleum Institute (API) in 1991. It
+is now the Petroleum Open Standards Consortium (POSC) [2]_ that has the
+stewardship. 
+
+When developing dlis standard, the main emphasis was put on easy writing. The
+files are structured in such a way that files can be written directly while
+acquiring the logs. This is very handy for the producers of the files, and
+equally tedious for the consumer that want to read them later on.
+Additionally is a very tolerant standard. It specifies general data-structures
+within the file, such as channels and frames, but allow the producers to modify
+these to fit their needs. It also allow for completely new structures, not
+defined by the standard itself, such as vendor-specific object-types. This all
+sums up to a fairly complex standard with a lot of quirks and weirdness to it.
+It is safe to say that dlis a particular difficult format to work with.
+
+Physical and logical files
+--------------------------
+
+The dlis standard distinguish between physical files and logical files. A
+physical file would be a .dlis file. A logical file on the other hand, is a
+collection of data that are logically connected. Typically this can be all
+measurements and metadata gathered at one run. A physical file may contain
+multiple logical files, where each logical file is completely independent of
+the other logical files:: 
+    
+                            physical file (.dlis)
+            --------------------------------------------------------
+           | logical file 0 | logical file 1 | ... | logical file n |
+            --------------------------------------------------------
+
+dlisio's main entrypoint, :py:func:`dlisio.load()` takes a physical file and
+returns a tuple-like object with all the logical files.
```

Comment:
Btw, not sure if we need to mention it here or in some other place, but it's worth mentioning that we do not yet support all the divisions/connections between physical and logical files, storage sets, file sets and all that other beautiful stuff.